src/js/_enqueues/wp/heartbeat.js

```diff
@@ -223,16 +223,33 @@
 				settings.checkFocusTimer = window.setInterval( checkFocus, 10000 );
 			}
 
-			$(window).on( 'unload.wp-heartbeat', function() {
+			$(window).on( 'pagehide.wp-heartbeat', function() {
```

Comment:
As with `pagehide.edit-post`, I don't see a single reference to the jQuery-namespaced `unload.wp-heartbeat` event name in WPdirectory, so I don't see any back-compat concerns here.